src/test/FixedPointMathLib.t.sol

```diff
@@ -216,20 +228,28 @@ contract FixedPointMathLibTest is DSTestPlus {
         assertEq(FixedPointMathLib.mulDivDown(x, y, denominator), (x * y) / denominator);
     }
 
-    function testFailMulDivDownOverflow(
+    function testMulDivDownOverflow(
         uint256 x,
         uint256 y,
         uint256 denominator
-    ) public pure {
-        // Ignore cases where x * y does not overflow or denominator is 0.
+    ) public {
+        // If the denominator or x is 0, the call should revert
```

Comment:
idt thsi comment is correct, x being 0 is fine

Reply:
i dont think adding this block his helpful tho, we check zero dominators dont work in the test below, trying to do too much here is no bueno

Reply:
Will assume denom != 0, muchos bueno

Reply:
Cleaned.